lib/sycamore/sycamore/data/element.py

```diff
@@ -191,10 +190,7 @@ def table(self) -> Optional[Table]:
     @table.setter
     def table(self, value: Optional[Table]) -> None:
         self.data["table"] = value
-        if value is not None:
-            self.data["text_representation"] = self.data["table"].to_csv()
-        else:
-            self.data["text_representation"] = None
+        self.data["text_representation"] = None
```

Comment:
Maybe a comment "invalidate cache"?

Could also `del self.data["text_representation"]` which may more clearly express the intent.

Reply:
I'd like to keep the assignment to avoid missing keys, added the comment.